File: src/validador.js
function validarEmail(email) {
  if (/@gmail\.com$/.test(email)) {
    console.log('Tá certo o e-mail');
  } else {
    console.log('Deu ruim!');
  }
}

function validarCep(cep) {
  if (/^\d{8}$/.test(cep)) {
    console.log('Tá correto teu CEP');
  } else {
    console.log('Tente de novo');
  }
}

function validarCpf(cpf) {
  if (/^\d{11}$/.test(cpf)) {
    console.log('Tá correto teu CPF');
  } else {
    console.log('Tente de novo');
  }
}

function validarTelefone(telefone) {
  if (/^\d{11}$/.test(telefone)) {
    console.log('Tá correto teu telefone');
  } else {
    console.log('Tente de novo');
  }
}

File: src/validador.py
# Conjunto de REGEX para validação de entradas.

def validar_email(email):
  if '@gmail.com' in email:
    print('Tá certo o e-mail')
  else:
   print('Deu ruim!')

def validar_cep(cep):
  if len(cep) == 8:
    print('Tá correto teu CEP')
  else:
    print('Tente de novo')

def validar_cpf(cpf):
  if len(cpf) == 11:
    print('Tá correto teu CPF')
  else:
    print('Tente de novo')


def validar_telefone(telefone):
  if len(telefone) == 11:
    print('Tá correto teu telefone')
  else:
    print('Tente de novo')

  
 